packages/client/tests/functional/relationMode/_matrix.ts

```diff
@@ -0,0 +1,64 @@
+import { defineMatrix } from '../_utils/defineMatrix'
+import { Providers } from '../_utils/providers'
+import { computeMatrix } from '../_utils/relationMode/computeMatrix'
+
+const RelationModeEnv = process.env.RELATION_MODE
+if (RelationModeEnv && RelationModeEnv !== 'prisma' && RelationModeEnv !== 'foreignKeys') {
```

Comment:
Shouldn't this also be able to be `''`?

Reply:
It is, we just check if it's defined it must be either `prisma` or `foreignKeys`